backend/app/services/web_scraper/__init__.py

```
@@ -0,0 +1,17 @@
+
+from app.services.web_scraper.scraper_service import (
+    ScrapedContent,
+    WebScraperService,
+    get_web_scraper_service,
+)
+
+__all__ = [
+    "WebScraperService",
+    "ScrapedContent",
+    "get_web_scraper_service",
+]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)